src/js/components/Select/SelectContainer.js

```diff
@@ -100,13 +100,15 @@ class SelectContainer extends Component {
       }
 
       // scroll to active option if it is below the fold
-      if (activeIndex >= 0) {
+      if (activeIndex >= 0 && selectNode) {
         const optionNode = this.optionsRef[activeIndex];
         const { bottom: containerBottom } = selectNode.getBoundingClientRect();
```

Comment:
A very minor point, but this should go within the test for `optionNode` since we don't use `containerBottom` outside of that context.

Reply:
true. let me change it.